src/effects/Distortion.cpp

```diff
@@ -296,14 +408,18 @@ std::unique_ptr<EffectUIValidator> EffectDistortion::PopulateOrExchange(
    {
       S.StartMultiColumn(4, wxCENTER);
       {
-         mTypeChoiceCtrl = S.Id(ID_Type)
+         mTypeChoiceCtrl = S
```

Comment:
ID_TYPE and other constants become unused, so may be removed from the header

Reply:
Oh, I see they are still used in `UpdateUIControls`.  That made me fear a bug -- that there was code still finding controls by id.  But no, there is a switch on ID that then just uses the pointers to controls saved in the validator.

So it is OK

Reply:
But ID_Type at least can go

Reply:
ok, removed